Makefile

```diff
@@ -161,7 +161,10 @@ test-full:
 	$(info log-file: test-$(TEST_SUFFIX).log)
 	PASSES="fmt build release unit integration functional e2e grpcproxy" ./test.sh 2<&1 | tee test-$(TEST_SUFFIX).log
 
-docker-test:
+ensure-docker-test-image-exists:
```

Comment:
Was this change intentional?

Reply:
Yes, I added it because otherwise CI would fail as someone had to push the image before (and folks with permissions should still do it to speed up CI). It's using https://github.com/etcd-io/etcd/pull/13487/commits/9404d523b821df463d2f0e33911ca0b9bcf0e51d